cartridge/roles/metrics.lua

```diff
@@ -138,6 +143,10 @@ local function apply_config(conf)
         end
         metrics_config_present = false
     end
+    metrics.clear()
+    metrics.enable_default_metrics(metrics_conf.include, metrics_conf.exclude)
+    metrics.enable_cartridge_metrics()
```

Comment:
Maybe cartridge metrics should be a section in default metrics so user will be able to include/exclude them too?

Reply:
Yes it should